datasync/chaindatafetcher/kas/repository_transactions.go

```diff
@@ -124,16 +122,17 @@ func transformToTxs(event blockchain.ChainEvent) []*Tx {
 
 		txs = append(txs, tx)
 	}
-	return txs
+	return txs, expiredEOAs
 }
 
 // InsertTransactions inserts transactions in the given chain event into KAS database.
 func (r *repository) InsertTransactions(event blockchain.ChainEvent) error {
-	txs := transformToTxs(event)
+	txs, expiredEOAs := transformToTxs(event)
 	if err := r.insertTransactions(txs); err != nil {
 		logger.Error("Failed to insertTransactions", "err", err, "blockNumber", event.Block.NumberU64(), "numTxs", len(txs))
 		return err
 	}
+	r.InvalidateCacheEOAList(expiredEOAs)
```

Comment:
`InvalidateCacheEOAList` can be called in a goroutine. I think network IO is too expensive for chainDataFetcher.

Reply:
Updated. e85b839b6de2c6396367fc23f4c9e1e648147ecc